crossdock/client/outboundttl/behavior.go

```diff
@@ -47,6 +47,11 @@ func Run(t crossdock.T) {
 	}, nil)
 	fatals.Error(err, "expected a failure for timeout")
 
+	if strings.HasPrefix(err.Error(), `BadRequest: unrecognized procedure "sleep/raw" for service "yarpc-test"`) {
```

Comment:
If we have to match on the Error string, what will our customers be forced to do? Is there anything better we could be doing here?